### ISSUE TITLE
xctest bug on iOS13.6

### ISSUE THREAD

**Issue by appl3s** [danielpaulus/go-ios#85]
When I started xctest in iOS 13.6, the code entered return. I think this is a bug, there may be such a method in iOS13.6 but there is no selector `_IDE_initiateControlSessionWithCapabilities` which causes xctest to not run normally
```go
func RunXCUIWithBundleIds(bundleID string, testRunnerBundleID string, xctestConfigFileName string, device ios.DeviceEntry, wdaargs []string, wdaenv []string) error {

	conn, err := dtx.NewConnection(device, testmanagerdiOS14)
	if err == nil {
                // here
		return runXUITestWithBundleIdsXcode12(bundleID, testRunnerBundleID, xctestConfigFileName, device, conn, wdaargs, wdaenv)
	}
	log.Debugf("Failed connecting to %s with %v, trying %s", testmanagerdiOS14, err, testmanagerd)

```

**Comment by appl3s** [comment]
Maybe this is also #68's answer

**Comment by appl3s** [comment]
change it to
```go
func RunXCUIWithBundleIds(bundleID string, testRunnerBundleID string, xctestConfigFileName string, device ios.DeviceEntry, wdaargs []string, wdaenv []string) error {
        // add version compare
        version, err := ios.GetProductVersion(device)
	if err != nil {
		return err
	}
	if version.Major() > 14 {
		conn, err := dtx.NewConnection(device, testmanagerdiOS14)
		if err == nil {
			return runXUITestWithBundleIdsXcode12(bundleID, testRunnerBundleID, xctestConfigFileName, device, conn, wdaargs, wdaenv)
		}
		log.Debugf("Failed connecting to %s with %v, trying %s", testmanagerdiOS14, err, testmanagerd)
	}

	conn, err := dtx.NewConnection(device, testmanagerd)
	if err != nil {
		return err
	}
```
maybe fix this problem

**Comment by danielpaulus** [comment]
Thanks! I will try this, so far my biggest issue has been getting access to different versions of devices to test this more. Last week I got myself a corellium.com account. Now I can make sure go-ios works with every ios version as it should. 
Thanks for figuring the issue out though @appl3s ! This is great :-D

**Comment by danielpaulus** [comment]
should be resolved with #87
